index.bs

```diff
@@ -3488,6 +3505,52 @@ While the best path forward may be to choose not to specify the feature,
 there is the risk that some implementations
 may ship the feature as a nonstandard API.
 
+<h3 id="multiple-algorithms">Write only one algorithm to accomplish a task</h3>
```

Comment:
Maybe "goal" instead of "task"? This immediately made me think of the event loop.